src/coreclr/vm/ceeload.cpp

```diff
@@ -4390,28 +4391,27 @@ BOOL Module::FixupNativeEntry(READYTORUN_IMPORT_SECTION* pSection, SIZE_T fixupI
 
 static LPCWSTR s_pCommandLine = NULL;
 
-// Retrieve the full command line for the current process.
-LPCWSTR GetManagedCommandLine()
-{
-    LIMITED_METHOD_CONTRACT;
-    return s_pCommandLine;
-}
+// Info for computing the command line if requested - owned by caller of CorHost2::ExecuteAssembly
```

Comment:
There is risk of dangling pointers.

These statics are going to be set even after `CorHost2::ExecuteAssembly` returns and the caller of `CorHost2::ExecuteAssembly` deletes them.